### PR TITLE
feat: print DTS generation time

### PR DESCRIPTION
**Pull request by Timeless0911**
## Summary

print DTS generation time

## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

File: packages/plugin-dts/src/dts.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { emitDts } from './tsc';
 import { ensureTempDeclarationDir, loadTsconfig } from './utils';
 
 export async function generateDts(data: DtsGenOptions) {
-  logger.info('Generating DTS...');
+  logger.ready('Generating DTS...');
```
Comment (chenjiahan) [inline packages/plugin-dts/src/dts.ts:9]: Is `start` more suitable here?

File: packages/plugin-dts/src/tsc.ts [hunk 2]
```diff
@@ -58,10 +62,10 @@ export function emitDts(
         logger.error(message);
       }
 
-      throw new Error('TypeScript compilation failed');
+      throw new Error('DTS generation failed');
```
Comment (fi3ework) [inline packages/plugin-dts/src/tsc.ts:65]: User `.d.ts.` instead of `DTS` in user side?

Comment (Timeless0911) [reply packages/plugin-dts/src/tsc.ts:65]: How about using `declarations files` instead of `DTS`?

Comment (Timeless0911) [reply packages/plugin-dts/src/tsc.ts:65]: we can add specific suffix file names after we support `autoExtensions` in DTS

Comment (fi3ework) [reply packages/plugin-dts/src/tsc.ts:65]: "DTS" does not seem to be a formal term. We can change it in the future based on how the log looks in the end.

**Review by fi3ework — APPROVED** [review]

**Pull request merged**